src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/annotatenuspec/AddVersionInfo.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The folder name for these files is a remnant from a previous direction I was going with this.  I'll change the folder name to something more aligned with these files' function.